pandas/io/sas/sas_xport.py

```diff
@@ -265,10 +265,6 @@ def __init__(
         else:
             # Copy to BytesIO, and ensure no encoding
             contents = filepath_or_buffer.read()
-            try:
-                contents = contents.encode(self._encoding)
```

Comment:
Is the encoding every applied to contents after this change? Does this matter?

Reply:
The encoding was only to get from text-mode to binary mode. As far as I can tell, SAS XPORT files don't have a text-mode, so the encoding is unnecessary in Python 3.

Later on, bytes corresponding to text columns are *de*coded into strings.

Reply:
Looks right.